ImportStatus.php

```diff
@@ -207,6 +208,14 @@ public function rateLimitReached($idSite)
         $this->saveStatus($status);
     }
 
+    public function cloudRateLimitReached($idSite, $errorMessage)
+    {
+        $status = $this->getImportStatus($idSite);
```

Comment:
@elabuwa Last thing, good to add test cases for this new status, rest all looks good
Example:
https://github.com/matomo-org/plugin-GoogleAnalyticsImporter/blob/4.x-dev/tests/Integration/ImportStatusTest.php#L577

https://github.com/matomo-org/plugin-GoogleAnalyticsImporter/blob/4.x-dev/tests/Integration/ImportStatusTest.php#L697

Reply:
new test added @AltamashShaikh